modules/gltf/src/lib/api/post-process-gltf.ts

```diff
@@ -310,13 +310,44 @@ class GLTFPostProcessor {
       const {ArrayType, byteLength} = getAccessorArrayTypeAndLength(accessor, accessor.bufferView);
       const byteOffset =
         (accessor.bufferView.byteOffset || 0) + (accessor.byteOffset || 0) + buffer.byteOffset;
-      const cutBufffer = buffer.arrayBuffer.slice(byteOffset, byteOffset + byteLength);
-      accessor.value = new ArrayType(cutBufffer);
+      let cutBuffer = buffer.arrayBuffer.slice(byteOffset, byteOffset + byteLength);
+      if (accessor.bufferView.byteStride) {
+        cutBuffer = this._getValueFromInterleavedBuffer(
+          buffer,
+          byteOffset,
+          accessor.bufferView.byteStride,
+          accessor.bytesPerElement,
+          accessor.count
+        );
+      }
+      accessor.value = new ArrayType(cutBuffer);
     }
 
     return accessor;
   }
 
+  /**
+   * Take values of particular accessor from interleaved buffer
+   * various parts of the buffer
+   * @param buffer
+   * @param byteOffset
+   * @param byteStride
+   * @param bytesPerElement
+   * @param count
+   * @returns
+   */
+  _getValueFromInterleavedBuffer(buffer, byteOffset, byteStride, bytesPerElement, count) {
```

Comment:
This does not need to be a method since it doesn't reference this. I think this helper could go into loaders.gl/math. 

I would call something like 
```
extractInterleavedBuffer<TArray>(...) {
}
```

It could be the counterpart of an `interleaveBuffers()` utility...